src/components/Select/index.tsx

```diff
@@ -3,38 +3,143 @@
  * This is licensed software from AccelByte Inc, for limitations
  * and restrictions contact your company contract manager.
  */
-
-import { default as AkSelect, SelectProps as AkSelectProps } from "@atlaskit/select";
+import React from "react";
+import {
+  default as AkSelect,
+  SelectProps as AkSelectProps,
+  AsyncSelect,
```

Comment:
I tried to use `ValidSelectAsync` to replace this functionality, but somehow the styling become so broken when implemented in `InlineSelect`, my guess is that because `ValidSelectAsync` is depend on `react-select-async-paginate` which has different structure/styling with components from `@atlaskit/select`. Therefore, `Select` from `@atlaskit/select` cannot work with `ValidSelectAsync` if they are combined into a single component.

Reply:
can we please at least create a separate AsyncSelect component? just to make this component small and simple since we don't have a lot of use case for async